### DISCUSSION TITLE
对话框上面的按钮可以默认显示文字吗?

### DISCUSSION THREAD

**Discussion by askgh** [ChatGPTNextWeb/NextChat#5630] [General]
看不清按键的含义
![图片](https://github.com/user-attachments/assets/ed37d028-5380-4f6d-860d-620a168966b1)

**Comment by lloydzhou** [comment]
对于经常用的人来说，会不会显得有点多余？

**Reply by askgh** [reply]
有设置可以配置就行,  你看dingding, 是不是图标都带了文字了, 不想猜, 这个图标具体是个什么含义

**Comment by askgh** [comment]
![图片](https://github.com/user-attachments/assets/dfe41f96-f901-435d-b5c9-86e57c5b8bcb)
这个地方, 得添加多种模型在这里, 是不是更宽一些, 现在的看不清到底加了多少

**Comment by askgh** [comment]
左侧对话框 列表, 有没有删除其他按钮, 或者是删除所有, 目前多个对话框, 比较杂乱
